docs/api/hosting-api.md

```diff
@@ -0,0 +1,99 @@
+
+Alright, let us open the form and give it a quick test:
+
+![A gif showing the option set working on a OneBlink form](../pics/OptionSetFormTest.gif)
+
+Eureka, it worked! Now we know how to set up this new Option Set.
+
+Let me present you with another situation though, what if, we had an error return INSTEAD of it succeeding? It is possible that when you write other end points, you will have error handling in your code that might throw when something happens, or another error could occur that you were not anticipating! We need to be able to check and understand what has gone wrong!
+Luckily, we have logs for the hosted API that you can view, so let's have a look at that now!
+
+## Viewing Logs and Viewing Metrics
+
+A keen eye would have seen in an earlier screenshot that these buttons existed on the API hosting page, which is where we will be going back to. We have both the Metrics Button and Logs button featured underneath the respective API's environment. If you had multiple environments, you would see metrics and logs buttons for each. First, let's look at the metrics page, so hit the metrics button for your hosted API!
+
+### Metrics
+
+You can see what the metrics screen looks like below:
+
+![An image showing the API Hosting metrics screen](../pics/ApiMetricsScreen.png)
+
+On this page, from top to bottom, you can see the named API and its environment in the top left of the page. You have 5 buttons on the right, reading 60 minutes, 6 hours, 24 hours, 7 days and 30 days from left to right for time periods you can select to view the metrics of.
+
+Additionally, you will see 3 boxes. These boxes will contain graphs or, if you're lucky enough to have no 4xx or 5xx HTTP responses, you will see a green tick with a nice message. If you are not lucky enough, these will also contain graphs that you can look at. 
+These metrics can provide insight into the traffic that you are receiving from your API. 
+
+### Logs
+
+You can see what the logs screen looks like below:
+
+![An image showing a Hosted API's log page with no logs](../pics/APIHostingLogsFrownyFace.png)
+
+From top to bottom again, you can see the API link with its environment in the top left then below that is a filter bar. 
+
+Here you can filter on a search parameter, a start time, and end time and a refresh button to search with those parameters. The search parameter will let you search your API logs for those parameters and it will return any that matches. The start date time and end date time will allow you to configure the time range that you would like to investigate logs for. 
+
+Below that, we have the area where logs show, in the screenshot above, you can see that it has a sad face with a message indicating that there were no logs found. You can change your time period to see if you can locate any logs. If you have been following long, you should have some from the call made earlier, so hit the start date and change it to yesterday, then hit refresh!
```

Comment:
```suggestion
Below that, we have the area where logs are show,n in the screenshot above, you can see that it has a sad face with a message indicating that there were no logs found. You can change your time period to see if you can locate any other logs. If you have been following along, you should have some from the call's made earlier, so hit the start date and change it to yesterday, then hit refresh!
```

Reply:
```suggestion
Below that, we have the area where logs are shown, in the screenshot above, you can see that it has a sad face with a message indicating that there were no logs found. You can change your time period to see if you can locate any other logs. If you have been following along, you should have some from the calls made earlier, so hit the start date and change it to yesterday, then hit refresh!
```

Lone n had snuck in there and it should be calls instead of call's, since it's multiple calls not possessive